includes/classes/class-omise-card-image.php

```diff
@@ -0,0 +1,198 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+if ( ! class_exists( 'Omise_Card_Image' ) ) {
+	class Omise_Card_Image {
+		/**
+		 * Compose the given parameters into the string of HTML &lt;img&gt; element
+		 * 
+		 * @param string $file Image file name with extension such as image.jpg
+		 * @param string $alternate_text Alternate text for the image
+		 * @return string HTML &lt;img&gt; element
+		 */
+		private static function get_image( $file, $alternate_text ) {
+			$url = WC_HTTPS::force_https_url( WC()->plugin_url() . '/assets/images/icons/credit-cards/' );
```

Comment:
@nimid I'm not sure about this one, what if some users still using `http` protocol, is it work for them?